docs/TypeScript.md

```diff
@@ -222,3 +200,35 @@ declare module WebdriverIO {
 }
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Tips and Hints
+
+### tsconfig.json example
+It's still good to have an example
```

Comment:
Is this sentence intentional?

Reply:
Haha, yes. However I agree that it makes no sense, removing.